arches_lingo/src/arches_lingo/themes/lingo_theme.ts

```diff
@@ -65,6 +99,14 @@ export const LingoPreset = definePreset(ArchesPreset, {
                             color: "{button-primary-background}",
                         },
                     },
+                    danger: {
+                        background: "{button-warn-active-background}",
+                        borderColor: "{button-warn-active-background}",
+                        hover: {
+                            background: "{button-warn-background}",
+                            borderColor: "{button-warn-background}",
+                        },
+                    },
```

Comment:
i get it, but it feels wrong to override a generic button theme with another button theme. What if we later want to add a `warn` button?